NEWS.md

```diff
@@ -1,9 +1,10 @@
-# EpiNow2 1.4.9000
+# EpiNow2 1.4.9001
 
 ## Documentation
 
 * Two new vignettes have been added to cover the workflow and example uses. By @sbfnk in #458 and reviewed by @jamesmbaazam.
 * Removed references to the no longer existing `forecast_infections` function. By @sbfnk in #460 and reviewed by @seabbs.
+* Added a GitHub Actions workflow to update the citation.cff file whenever the DESCRIPTION is changed. By @jamesmbazam and reviewed by @seabbs.
```

Comment:
Needs to be updated to reflect clarifications in the PR conversation.

Reply:
+ to tag in @Bisaloo and @sbfnk

Reply:
I've updated it.